test/size.txt

```diff
@@ -1,45 +1,47 @@
-255.53 kB   |   56.62 kB   |   20.46 kB   |   v0/amp-analytics-0.1.js
- 90.15 kB   |    8.66 kB   |    3.39 kB   |   v0/amp-anim-0.1.js
-  88.6 kB   |    6.93 kB   |    2.83 kB   |   v0/amp-audio-0.1.js
- 81.97 kB   |    7.92 kB   |    3.03 kB   |   v0/amp-brid-player-0.1.js
-105.55 kB   |    7.78 kB   |    2.95 kB   |   v0/amp-brightcove-0.1.js
-212.04 kB   |   33.13 kB   |    9.98 kB   |   v0/amp-carousel-0.1.js
- 74.56 kB   |    6.51 kB   |    2.64 kB   |   v0/amp-dailymotion-0.1.js
- 93.06 kB   |       5 kB   |    2.17 kB   |   v0/amp-dynamic-css-classes-0.1.js
-146.44 kB   |   15.59 kB   |    6.19 kB   |   v0/amp-facebook-0.1.js
- 35.03 kB   |    6.09 kB   |    2.47 kB   |   v0/amp-fit-text-0.1.js
-101.36 kB   |    8.11 kB   |     3.2 kB   |   v0/amp-font-0.1.js
-128.44 kB   |   10.79 kB   |    4.15 kB   |   v0/amp-form-0.1.js
- 85.15 kB   |    7.72 kB   |    2.93 kB   |   v0/amp-fx-flying-carpet-0.1.js
-142.71 kB   |   14.22 kB   |    5.45 kB   |   v0/amp-iframe-0.1.js
-229.23 kB   |   33.21 kB   |   10.31 kB   |   v0/amp-image-lightbox-0.1.js
- 99.38 kB   |    7.57 kB   |    2.95 kB   |   v0/amp-instagram-0.1.js
- 85.49 kB   |    8.27 kB   |    3.39 kB   |   v0/amp-install-serviceworker-0.1.js
-  81.4 kB   |    7.39 kB   |    2.91 kB   |   v0/amp-jwplayer-0.1.js
-110.87 kB   |    8.44 kB   |    3.28 kB   |   v0/amp-kaltura-player-0.1.js
-139.67 kB   |   12.73 kB   |    4.42 kB   |   v0/amp-lightbox-0.1.js
- 86.77 kB   |    6.02 kB   |    2.56 kB   |   v0/amp-list-0.1.js
-152.02 kB   |    13.6 kB   |    4.99 kB   |   v0/amp-live-list-0.1.js
-145.88 kB   |   39.86 kB   |   14.26 kB   |   v0/amp-mustache-0.1.js
-125.91 kB   |   20.53 kB   |    6.07 kB   |   v0/amp-pinterest-0.1.js
- 73.67 kB   |    6.17 kB   |    2.48 kB   |   v0/amp-reach-player-0.1.js
- 91.28 kB   |   10.49 kB   |    3.72 kB   |   v0/amp-sidebar-0.1.js
-108.54 kB   |   11.43 kB   |    4.27 kB   |   v0/amp-slides-0.1.js
-105.84 kB   |   11.26 kB   |    4.25 kB   |   v0/amp-social-share-0.1.js
- 74.25 kB   |     6.3 kB   |    2.53 kB   |   v0/amp-soundcloud-0.1.js
- 82.22 kB   |    8.08 kB   |    3.01 kB   |   v0/amp-springboard-player-0.1.js
- 86.22 kB   |    7.11 kB   |    2.84 kB   |   v0/amp-sticky-ad-0.1.js
-146.91 kB   |   15.71 kB   |    6.23 kB   |   v0/amp-twitter-0.1.js
-112.08 kB   |   10.13 kB   |    3.89 kB   |   v0/amp-user-notification-0.1.js
- 74.17 kB   |    6.26 kB   |    2.53 kB   |   v0/amp-vimeo-0.1.js
- 73.71 kB   |    6.13 kB   |    2.49 kB   |   v0/amp-vine-0.1.js
-114.22 kB   |    8.83 kB   |    3.45 kB   |   v0/amp-youtube-0.1.js
-158.31 kB   |   34.82 kB   |   12.04 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+      max   |         min   |       gzip   |   file
+      ---   |         ---   |        ---   |   ---
+ 57.48 kB   |     5.94 kB   |    2.48 kB   |   alp.js / alp.max.js
+659.67 kB   |   128.96 kB   |    38.5 kB   |   shadow-v0.js / amp-shadow.js
```

Comment:
nvm, if i'm understanding correctly, this is correct as it is an alternative to the main binary.